packages/bpk-component-button/README.md

```diff
@@ -24,6 +24,7 @@ export default () => (
     <BpkButton large>Large primary</BpkButton>
     <BpkButton secondary>Secondary</BpkButton>
     <BpkButton link>Link</BpkButton>
+    <BpkButton outline>Link</BpkButton>
```

Comment:
Should this be worded Link or rather 'Outline' as the secondary and primary above?

Reply:
Yep, good catch